Rakefile

```diff
@@ -8,6 +10,12 @@ Rake::TestTask.new(:test) do |t|
   t.test_files = FileList['test/**/*_test.rb']
 end
 
-RuboCop::RakeTask.new
+Reek::Rake::Task.new do |task|
+  task.fail_on_error = false
+end
+
+RuboCop::RakeTask.new do |task|
+  task.fail_on_error = false
+end
 
-task default: :test
+task default: [:rubocop, :reek, :test]
```

Comment:
Default Rake task (invoked via `bin/rake`) now runs Rubocop, Reek, and Minitest.